src/main/java/org/jabref/gui/strings/StringDialogViewModel.java

```diff
@@ -0,0 +1,85 @@
+package org.jabref.gui.strings;
+
+import java.util.Optional;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import javafx.beans.property.ListProperty;
+import javafx.beans.property.ObjectProperty;
+import javafx.beans.property.SimpleListProperty;
+import javafx.beans.property.SimpleObjectProperty;
+import javafx.beans.property.SimpleStringProperty;
+import javafx.beans.property.StringProperty;
+import javafx.collections.FXCollections;
+
+import org.jabref.gui.AbstractViewModel;
+import org.jabref.logic.bibtex.comparator.BibtexStringComparator;
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.entry.BibtexString;
+
+public class StringDialogViewModel extends AbstractViewModel {
+
+    private final ListProperty<StringViewModel> allStrings = new SimpleListProperty<>(FXCollections.observableArrayList());
+    private final ObjectProperty<StringViewModel> selectedItemProperty = new SimpleObjectProperty<>();
+    private final StringProperty newStringLabelProperty = new SimpleStringProperty("");
+    private final BibDatabase db;
+
+    public StringDialogViewModel(BibDatabase bibDatabase) {
+        this.db = bibDatabase;
+        addAllStringsFromDB();
+
+    }
+
+    private void addAllStringsFromDB() {
+
+        Set<StringViewModel> strings = db.getStringKeySet()
+                                         .stream()
+                                         .map(string -> db.getString(string))
```

Comment:
Why not directly get a collection of all strings from the database (is there a method like `getStringValueSet`? Pesonally, I would like to have only a method that returns the strings in the form of a list of `BibtexString`s)